src/modules/scoresMethods.js

```diff
@@ -0,0 +1,23 @@
+const easterEggQuest = {
+  async addScore(user, score) {
+    const idUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/x7gdzdyghc93DAWobtyz/scores/';
+    const response = await fetch(idUrl, {
+      method: 'POST',
+      body: JSON.stringify({
+        user,
+        score,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+    return response.json();
+  },
+  async getScores() {
+    const idUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/x7gdzdyghc93DAWobtyz/scores/';
+    const response = await (await fetch(idUrl)).json();
+    return response.result;
+  },
+};
+
+export default easterEggQuest;
```

Comment:
- [ ] Its great that you have implemented the functionality to add data to the API :blush: but when i click on the refresh button then it doesn't give me the new data which i have added kindly fix it :+1:
- [ ] Your page is not supposed to reload by clicking on the Refresh button of data. Kindly get the data without refreshing the page while clicking on the Refresh button :+1:

Reply:
Hello, @HAMMAS-SALEEM  Thanks for the review I can now improve my code. :+1: 
It was my fault for not telling you that it was a top 10, and the Maximus score was 100, you submitted 101 twice; now I know. :smile:
But I fixed it now the leaderboard accepts 1000 participants and the Maximus score is 1000. :100: 
Now the page displays the leaderboard and doesn't reload the URL.